sql/core/src/test/resources/tpch/q1.sql

```diff
@@ -0,0 +1,23 @@
+-- using default substitutions
+
+select
+	l_returnflag,
```

Comment:
Can we use space instead of `tab`?

Reply:
This is the generated SQL. I plan to keep them unchanged.